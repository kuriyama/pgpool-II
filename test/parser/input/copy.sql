COPY t FROM 'xxx';
COPY t FROM stdin;
COPY t(x,y) FROM 'xxx';
COPY t FROM 'xxx' BINARY;
COPY t FROM 'xxx' OIDS;
COPY t FROM 'xxx' DELIMITER ',';
COPY t FROM 'xxx' NULL 'x';
COPY t TO 'yyy';
COPY t TO stdout;
COPY t(x,y) TO 'yyy';
COPY t TO 'yyy' BINARY;
COPY t TO 'yyy' OIDS;
COPY t TO 'yyy' DELIMITER ',';
COPY t TO 'yyy' NULL 'x';
COPY (SELECT * FROM t) TO 'xxx';
