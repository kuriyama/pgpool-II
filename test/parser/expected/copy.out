COPY "t" FROM 'xxx'
COPY "t" FROM STDIN
COPY "t"("x", "y")  FROM 'xxx'
COPY "t" FROM 'xxx' BINARY 
COPY "t" FROM 'xxx' OIDS 
COPY "t" FROM 'xxx' DELIMITERS ','
COPY "t" FROM 'xxx' NULL 'x' 
COPY "t" TO 'yyy'
COPY "t" TO STDOUT
COPY "t"("x", "y")  TO 'yyy'
COPY "t" TO 'yyy' BINARY 
COPY "t" TO 'yyy' OIDS 
COPY "t" TO 'yyy' DELIMITERS ','
COPY "t" TO 'yyy' NULL 'x' 
COPY ( SELECT "*" FROM "t") TO 'xxx'
